docs/usage/configuration-options.md

```diff
@@ -2630,6 +2630,12 @@ Be careful with remapping `warn` or `error` messages to lower log levels, as it
 
 Add to this object if you wish to define rules that apply only to major updates.
 
+## maxMajorIncrement
+
+This is particularly useful for preventing upgrades from [SemVer](https://semver.org/)-based versions to [CalVer](https://calver.org/)-based ones.
+
+For example, if you're on version `19.2.0`, Renovate will upgrade to `21.0.0` but filter out `2023.3.3`.
+
```

Comment:
Please also add an example.

Reply:
@RahulGautamSingh, I actually had included an example initially, but then reconsidered.

I don’t think this option needs an example, since it’s not something users are expected to tweak often. Including one might unintentionally suggest it’s common or recommended to change it.

For the few advanced users who will need to adjust this, the current explanation in the documentation should be clear enough.

However, if you still think I should add an example, please let me know and I'll readily do it. :)

Reply:
I think an example of using it with `packageRules` to tweak behavior for a dep will be the use-case for most adv. users. So, that can be added.